scipy/sparse/_extract.py

```diff
@@ -93,8 +93,8 @@ def tril(A, k=0, format=None):
            [4, 0, 0, 0, 0],
            [0, 0, 0, 0, 0]])
     >>> tril(A, format='csc')
-    <3x5 sparse array of type '<class 'numpy.int32'>'
-            with 4 stored elements in Compressed Sparse Column format>
+    <3x5 sparse matrix of type '<class 'numpy.int32'>'
```

Comment:
```suggestion
    <3x5 sparse array of type '<class 'numpy.int32'>'
```

Reply:
This was fixed after resolving the merge conflict. Thank you Lucas.